### PR TITLE
fix: aut height css tweaks

### PR DESCRIPTION
**Pull request by marktnoonan**
This moves the global CSS into the Runner.vue page and uses an `is-screenshotting` class on the document body instead of `v-bind` to simplify things, as well as sidestep some Vue component lifecycle behavior that can be tricky to debug and was causing screenshots to be taken right before the style updated.

We can come up with a better convention than doing this where I have it in this PR, but screenshots look good, system tests pass, I think this is OK for unblocking the squished height issue.

### CODE REVIEW

File: packages/app/src/pages/Specs/Runner.vue [hunk 1]
```diff
@@ -72,3 +75,50 @@ window.__showSelectorPlaygroundForTestingPurposes = () => {
...
+  position: relative;
+    margin: 0 auto;
+}
+
+.is-screenshotting #unified-runner {
+    margin: unset;
+}
+
+#unified-runner > .screenshot-height-container {
+  height: 100%
```
Comment (tgriesser) [inline packages/app/src/pages/Specs/Runner.vue:117]: Should this be dynamic based on the user's defined screen height?

Comment (tgriesser) [reply packages/app/src/pages/Specs/Runner.vue:115]: We can do this as a separate PR since this one is mainly to unblock develop, just wanted to note it

Comment (marktnoonan) [reply packages/app/src/pages/Specs/Runner.vue:115]: Do you mean viewport height? If so there's a `viewportStyle` implemented separately that handles that. This rule is only targeting the wrapper div that sometimes, for screenshots, needs to handle height differently.

Comment (tgriesser) [reply packages/app/src/pages/Specs/Runner.vue:115]: Ah gotcha

**Review by ZachJW34 — APPROVED** [review]

**Review by tgriesser — APPROVED** [review]

**Pull request merged**